sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/Snapshot.java

```diff
@@ -36,6 +38,23 @@
     /** @return the details of the source from which snapshot is created */
     CreationSource source();
 
+    /**
+     * Gets the percentage complete for the background copy when a resource is created via the CopyStart operation.
+     * <p>For latest progress,{@link Snapshot#refresh()} or {@link Snapshot#refreshAsync()} should be called prior to this method.</p>
```

Comment:
Is it possible to be null? If yes, maybe describe that as well.

Reply:
added in @return